releng/download/releases/9.4/compositeArtifacts.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?compositeArtifactRepository version='1.0.0'?>
+<repository name='CDT 9.4 Update Site'
+    type='org.eclipse.equinox.internal.p2.artifact.repository.CompositeArtifactRepository' version='1.0.0'>
+  <properties size='1'>
+    <property name='p2.timestamp' value='1673888544650'/>
+  </properties>
+  <children size='5'>
```

Comment:
Shouldn't size be 2 ?

Reply:
As in, not 5.

Reply:
Yes. Thanks for the thorough review.

Reply:
See #248

Reply:
Great, thank you very much for fixing this! :)